vignettes/diseasy-observables.Rmd

```diff
@@ -77,33 +63,35 @@ obs$describe()
 # Getting observations
 We can query the model to give the data for a given observable in a given time frame:
 ```{r example_1, eval = suggests_available}
-obs$get_observation(observable = "n_population",
-                    start_date = as.Date("2020-03-01"),
-                    end_date = as.Date("2020-05-01"))
-
-# NOTE: "target_conn" is a SQLite data base which has "numeric" as the "date" type.
-# NOTE: The population here is not stratified, so covers all countries in the Google data
+obs$get_observation(
+  observable = "n_positive",
+  start_date = as.Date("2020-03-01"),
+  end_date = as.Date("2020-04-30")
+)
 ```
 
 If we want to stratify our data, we supply the stratification argument.
 This argument is designed to be flexible, but it means they need to be wrapped in `rlang::quos()`.
 We will see below, why that is.
 ```{r example_2, eval = suggests_available}
-obs$get_observation(observable = "n_hospital",
-                    stratification = rlang::quos(age_group),
-                    start_date = as.Date("2020-03-01"),
-                    end_date = as.Date("2020-05-01"))
+obs$get_observation(
+  observable = "n_hospital",
+  stratification = rlang::quos(age),
```

Comment:
Consider cutting in age groups.

Reply:
Thanks. I just looked at it, and I think it is better to not do it. 
The idea is that we in the vignette gradually increase complexity:

first: No stratification
secondly: Stratification by an existing feature
thirdly: Stratification by an expression / computation using existing features

If anything, we should use cut in the third example instead of "age < 20"